languages/fr-fr.js

```diff
@@ -0,0 +1,181 @@
+module.exports.momentLocale = 'fr'
+
+module.exports.strings = {
+  _NOT_LOCALIZED: key => `${key} n'a pas encore été traduite en français.`,
+  ADMIN_CLEAR_WISHLISTS_BUTTON: `Effacer toutes les listes d'envies`,
+  ADMIN_CLEAR_WISHLISTS_DESCRIPTION: `Cela supprimera instantanément et de manière <b>irréversible toutes les listes d'envies!</b> Pensez à faire une sauvegarde de la base de données avant d'utiliser cette option.`,
+  ADMIN_CLEAR_WISHLISTS_HEADER: `Suppression de la liste d'envie`,
+  ADMIN_SETTINGS_CLEARDB_BUTTON: `Videz la liste d'envie`,
+  ADMIN_SETTINGS_CLEARDB_DESCRIPTION: '<b>Attention</b>: Ces options <b>supprime les données</b>! Pensez à faire une sauvegarde avant d\'utiliser cette option.',
+  ADMIN_SETTINGS_CLEARDB_HEADER: 'Suppression des données',
+  ADMIN_SETTINGS_CLEARDB_SUCCESS: `Suppression de toutes les listes d'envies.`,
+  ADMIN_SETTINGS_HEADER: 'Paramètres Administrateurs',
+  ADMIN_SETTINGS_USERS_ADD_BUTTON: 'Ajouter un Utilisateur',
+  ADMIN_SETTINGS_USERS_ADD_HEADER: 'Ajouter un utilisateur',
+  ADMIN_SETTINGS_USERS_ADD_PLACEHOLDER: 'jean',
+  ADMIN_SETTINGS_USERS_ADD_USERNAME: `Nom d'utilisateur`,
+  ADMIN_SETTINGS_USERS_EDIT_DELETE_FAIL_ADMIN: `Échec de la suppression : l'utilisateur est un administrateur.`,
+  ADMIN_SETTINGS_USERS_EDIT_DELETE_SUCCESS: name => `Utilisateur supprimé avec succès ${name}`,
+  ADMIN_SETTINGS_USERS_EDIT_DEMOTE_NOT_ADMIN: `l'utilisateur n'est pas un administrateur`,
+  ADMIN_SETTINGS_USERS_EDIT_DEMOTE_SELF: 'Vous ne pouvez pas vous supprimer vous même.',
```

Comment:
I understood it as a user being an admin and not being allowed to remove the admin role from his own user.
In this case it would be more : `Vous ne pouvez pas vous rétrograder vous-même.`